misc/working_w_l10n_agency.md

```diff
@@ -0,0 +1,88 @@
+* Account Payable form that contains banking information, billing cycle (monthly, quarterly or as soon as) and payment terms (within 30 or 45 days); billing email address.
+* An official rate sheet that includes the following but not limited to:
+** Word rate for each of the interested locales as in new words, 100% match, fuzzy match for translation, review, proofreading
+** Hourly rate: engineering, QA and testing, PM rate
+** Minimum charge for a project and the thresholds by number of words per locale
+** Rush rate
+
+PO, or Purchase Order, may or may not be necessary in order to get vendor service. Some teams have ongoing need for l10n service and have a budget set aside annually. This is a good reason to have a PO established. For most projects that are one time need, or sporadic, it’s best to go without a PO, and bill to the serviced department as needed.
+
+## Key contacts and responsibilities:
+
+** Account Manager: relationship, escalation of issues; complaints about quality or deadlines
+** Project Manager: handles day to day operation. Coordinates resources on the agency side; provides estimate of a project's deadline and cost
+** Account Payable Manager: sends the bill to Mozilla Account Payable
+
+## Set up accounts:
+
+Depending on the needs and where the source file can be accessed, it’s good to have an account set up in the following places:
+
+* GitHub: usually for content heavy type of projects, such as legal document; can commit directly
+* Pontoon: as translator when there is no community; or no account when there is an existing community (the vendor will provide suggestions only, the community will approve them)
+* Pootle: as translator when there is no community; or no account when there is an existing community (the vendor will provide suggestions only, the community will approve them)
+* Bugzilla: inform which product group l10n bugs are under
+* Google drive: to view, review, and translate directly
+* If WordPress platform is used, sign up an account. It’s project specific
+
+Once accounts are set up, allow the agency to explore the tools, flows and the product features to familiarize itself before actual work starts. Setting up a meeting to share your screen might be a good idea, and use this opportunity to answer their questions about the project or the tools.
+
+## Localizing a project
+
+### Planning:
+
+Once outsourcing is a necessity for a project, before reaching out to the l10n agency to make an official requests, make sure to check the following:
+
+* Required languages
+* Scope of work: other than translation; review; testing
+* Milestones and Deadline: negotiate it so the agency has sufficient time to finish each of the required task, and the community has enough time to review and sign off.
+* Check with communities to get their buy-ins on languages to outsource and tasks to divide among the contributing parties
+
+#### Communications:
```

Comment:
This is still part of the planning phase. Since the list is a bit long, I feel the need of calling it out separately.